frontend/ui/impl/src/main/resources/bug_report.md

```diff
@@ -3,31 +3,15 @@
 - IDE - %ide%
 - Operating system - %os%
 
-## IDEA - stack trace
-%stacktraces%
-
-## IDEA - additional info
-%additionalInfo%
-
 ## Bug description
 **Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behaviour**.
 
-## (optional) Logs
```

Comment:
It's pretty unlikely that anybody will really include these... esp. that you need to collect them BEFORE the issue occurs 😅

Reply:
I agree but - we do have this description somewhere in developlment.md, dont we? It may be useless in this scope but still handy for devs

Reply:
<img width="987" alt="image" src="https://user-images.githubusercontent.com/3383210/211917971-58b7069a-88a4-437d-87e4-195077ca89e5.png">